### ISSUE TITLE
Does not handle input type

### ISSUE THREAD

**Issue by aboe026** [jakubfiala/graphql-json-schema#1]
transform breaks if there is [input type](http://graphql.org/learn/schema/#input-types) in the schema. I was able to fix this locally by modifying `toSchemaProperty` as follows:
```
const toSchemaProperty = field => {
  let propertyType = getPropertyType(field.type);

  if ('$ref' in propertyType) propertyType = { allOf: [propertyType, { title: field.name.value }] };

  if (!!field.arguments) {
    return Object.assign(propertyType, {
      title: field.name.value,
      arguments: field.arguments.map(a => {
        return {
          title: a.name.value,
          type: getPropertyType(a.type),
          defaultValue: a.defaultValue
        };
      })
    });
  } else {
    return Object.assign(propertyType, {
      title: field.name.value
    });
  }
}
```
there's probably a nicer way to do it though.

**Comment by jakubfiala** [comment]
@aboe026  thanks for flagging this up, I've fixed this in #2 and will release a new version of the package now.

FYI Inputs will now have an "input" flag set to `true`, so you can check what they are (see `/spec/data` to see what the JSON looks like)

**Comment by aboe026** [comment]
perfect thank you!
